proximal/examples/test_deconv_sinv_psf.py

```diff
@@ -1,5 +1,9 @@
+# Override the default ProxImaL installation in the system
+import sys
+sys.path.append('../../')
```

Comment:
Overriding the ProxImaL installed in your system with the most recent version. Equivalent to `pip install --local -e .`